environment-docs.yml

```diff
@@ -1,9 +1,9 @@
 name: xhydro_lstm
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - python >=3.12,<3.13
+  - tensorflow >=2.10
```

Comment:
Are we going to be running `tensorflow` over the notebooks/examples on ReadTheDocs?

Reply:
yes, but not the GPU version, just CPU